backend/src/statistics/tests/statistics.e2e.spec.ts

```diff
@@ -0,0 +1,155 @@
+import { INestApplication } from "@nestjs/common";
+import { Test } from "@nestjs/testing";
+import { Language } from "@prisma/client";
+import { LoggerModule } from "nestjs-pino";
+import request from "supertest";
+
+import { AuthModule } from "src/auth/auth.module";
+import { MockSessionMiddleware } from "src/common/middlewares/test/MockSessionMiddleware";
+import { PrismaServiceModule } from "src/core/prisma.service.module";
+import { TestClient } from "src/core/test/test-client";
+import { RedisServiceModule } from "src/redis/redis.service.module";
+
+import { StatisticsModule } from "../statistics.module";
+
+import { Difficulty } from "~shared/types/base";
+import { Status } from "~shared/types/base";
+
+const userFixtures = [
+  {
+    id: 1,
+    email: "janedoe@email.com",
+    name: "Jane Doe",
+    password: "password",
+  },
+  { id: 2, email: "johndoe@email.com", name: "John Doe", password: "password" },
+];
+
```

Comment:
@dextertanyj can i check how do you link the fixtures using the foreign key in object code? The test for this file is not working yet

Reply:
You can take a look at either 
- https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#nested-writes
- https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#connect-an-existing-record